### DISCUSSION TITLE
Kuei Jin actor sheet option?

### DISCUSSION THREAD

**Discussion by Azhurai** [JohanFalt/Foundry_WoD20#896] [Q&A] [answered]
I have some players who wanted to play as Kuei jin, but I have no idea how to set that up as there isn't a Kuei jin option. 

If one could be added in the future that would be appreciated, but for now are there any workarounds or ways to add a custom actor sheet type?

**Comment by JohanFalt** [comment]
There will not be support for those in a long time so the solution for you is not really writhing the system itself.

Instead I would look at the module that makes it possible to add editable pdf files as sheets. I know it is discussed in the Foundry discord channel for "Other game systems". 

I'll see how it works and write back.

**Reply by Azhurai** [reply]
Okie dokie thanks for the advice

**Comment by JohanFalt** [comment, marked as answer]
The module is called PDF Pager and what I did was took an editabel character sheet from MrGone's webpage, placed it as a file in the world then linked that sheet to a Journal Entry - and now you have a sheet at least you can use.

**Reply by Azhurai** [reply]
Yeah I found that, only issue now is linking the Abilities and attributes to be able to be rolled

**Reply by JohanFalt** [reply]
Not sure you can thus you need to let them use the free dices in the WoD20 (the D10 image)
